doc/dev/mgmt/tests.md

```diff
@@ -10,42 +10,40 @@
     -   [Running tests in live mode](#running-tests-in-live-mode)
 - [Using the Azure Python SDK test framework](#using-the-azure-python-sdk-test-framework)
 
-IMPORTANT NOTE: All the commands in this page assumes you have loaded the [dev_setup](https://github.com/Azure/azure-sdk-for-python/blob/master/doc/dev/dev_setup.md) in your currently loaded virtual environment.
+IMPORTANT NOTE: All the commands in this page assumes you have loaded the [dev_setup][dev_setup] in your currently loaded virtual environment.
 
 # Overview
 
-This page is to help you write tests for Azure Python SDK when these tests require Azure HTTP requests. The Azure SDK test framework uses the `azure-devtools` package, which in turn rests upon on a HTTP recording system ([vcrpy](https://pypi.python.org/pypi/vcrpy)) that enables tests dependent on network interaction to be run offline.
+This page is to help you write tests for Azure Python SDK when these tests require Azure HTTP requests. The Azure SDK test framework uses the [`azure-devtools`][azure_devtools] package, which in turn rests upon on a HTTP recording system ([vcrpy][vcrpy]) that enables tests dependent on network interaction to be run offline.
 
 In this document, we will describe:
--   How to run the tests offline (using previously recorded HTTP interactions)
--   How to run the tests online (by authenticating with Azure to record new HTTP interactions)
--   How to write new tests using our utility classes
+-   [How to run the tests online (by authenticating with Azure to record new HTTP interactions)](#running-tests-in-live-mode)
+-   [How to run the tests offline (using previously recorded HTTP interactions)](#running-tests-in-playback-mode)
+-   [How to write new tests using our utility classes](#using-the-azure-python-sdk-test-framework)
```

Comment:
not sure if this should be showing a hyperlink to the sections like the two above, but it doesn't.